org.mwc.debrief.multipath2/src/org/mwc/debrief/multipath2/MultiPathView.java

```diff
@@ -91,7 +87,9 @@ public MultiPathView()
 	 */
 	public void createPartControl(final Composite parent)
 	{
-		_ui = new MultiPathUI(parent, SWT.EMBEDDED);
+	  parent.setLayout(new FillLayout());
+		_ui = new MultiPathUI(parent, SWT.EMBEDDED);  
+		// SWT.NONE allows some controls to appear - but not graph

```

Comment:
With this set to SWT.NONE I get this image:
![somecontrols](https://cloud.githubusercontent.com/assets/1108513/16225522/ddae20fe-379e-11e6-97cb-f416166e27af.png)

With it set to SWT.EMBEDDED I get a blank RCP View.

SWT.Embedded does seem to be unit specific: http://www.eclipsezone.com/eclipse/forums/t29500.html